lib/useDataApi.js

```diff
@@ -18,12 +18,20 @@ const dataFetchReducer = (state, action) => {
                 isLoading: false,
                 isError: true,
             };
+        case 'FETCH_RETRY':
+            return {
+                ...state,
+                isLoading: false,
```

Comment:
Shouldn't that be loading when retrying?

Reply:
Oh it's old code, I remove it